generator/stdlib/src/commonMain/kotlin/com/github/jcornaz/kwik/generator/stdlib/Strings.kt

```diff
@@ -10,84 +10,45 @@ import kotlin.random.Random
 /**
  * Returns a generator of String of length between [minLength] and [maxLength] (inclusive)
  *
- * @param charset Set of character to be used in generated strings
- * @param exclude Characters to exclude from generated strings
+ * @param charGenerator A generator of characters which will be used to construct the strings
  */
 fun Generator.Companion.strings(
```

Comment:
Can you leave the old function signatures? Their implementation should be a simple delegation to the new ones:

Example: 

```kotlin
fun Generator.Companion.strings(
  minLength: Int = 0,
  maxLength: Int = maxOf(minLength, KWIK_DEFAULT_MAX_SIZE),
  charset: Set<Char> = CharSets.printable,
  exclude: Set<Char> = emptySet(),
) = strings(minLength, maxLength, characters(charset, exclude))
```

Maybe I'll deprecate them or maybe I'll keep the overlaod. But either way it is nice to not *force* the user to use the new API right away.

Reply:
Thanks, I completely forgot about backwards compatability. I will restore the signatures. Should I restore the old tests as well?